### PR TITLE
Fix reflection configuration for native-image

### PR DESCRIPTION
**Pull request by andreaTP**
An image containing this fix is currently deployed in the Operate First environment.
This fixes both the validation and the download from URL that happened to be broken in the `native-image` build.

Please check if anything is missing!

### CODE REVIEW

**Comment by EricWittmann** [comment]
LGTM WDYT @carlesarnal ?

File: app/tools/extractRegisterForReflection.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS org.reflections:reflections:0.10.2
+//DEPS io.apicurio:apicurio-data-models:1.1.26
+
+import java.util.*;
```
Comment (carlesarnal) [inline app/tools/extractRegisterForReflection.java:5]: I like to avoid starts for imports, but it's just a personal thing.

**Review by carlesarnal — APPROVED** [review]
Apart from the small comment about imports, looks good to me.

File: app/tools/extractRegisterForReflection.java (hunk 1 above)
Comment (andreaTP) [reply app/tools/extractRegisterForReflection.java]: fixed!

**Pull request merged**